Firestore/Source/Local/FSTLRUGarbageCollector.mm

```diff
@@ -103,10 +103,11 @@ - (LruResults)collectWithLiveTargets:(NSDictionary<NSNumber *, FSTQueryData *> *
   size_t currentSize = [self byteSize];
   if (currentSize < _params.minBytesThreshold) {
     // Not enough on disk to warrant collection. Wait another timeout cycle.
-    LOG_DEBUG("Garbage collection skipped; Cache size %i is lower than threshold %i", currentSize,
-              _params.minBytesThreshold);
+    LOG_DEBUG("Garbage collection skipped; Cache size %s is lower than threshold %s",
+              std::to_string(currentSize), std::to_string(_params.minBytesThreshold));
```

Comment:
These explicit conversions are unnecessary. The only /specifier/ we support is `%s`, but it still accepts any value.

I should really change that to a python-style `{}` for the specifier so it's clearer.

Reply:
Removed.